server/routerlicious/packages/routerlicious-base/src/alfred/services/deltaService.ts

```diff
@@ -0,0 +1,129 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { toUtf8 } from "@fluidframework/common-utils";
+import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
+import { ISequencedOperationMessage, ITenantManager, MongoManager } from "@fluidframework/server-services-core";
+
+export class DeltaService {
+    constructor(
+        protected readonly mongoManager: MongoManager,
+        protected readonly tenantManager: ITenantManager,
+    ) {
+    }
```

Comment:
minor nit, feel free to ignore: the pattern I think I noticed throughout the codebase is to have both `{` and `}` on the same line for empty constructors. Example:
```ts
constructor(
        protected readonly mongoManager: MongoManager,
        protected readonly tenantManager: ITenantManager,
    ) { }
```

Reply:
addressed in [84636d2](https://github.com/microsoft/FluidFramework/pull/11008/commits/84636d2e2ccf4c1e6b6eed0b0ca01dd2c6b9615f)